packages/paste-website/src/pages/icons/how-to-add-an-icon/index.mdx

```diff
@@ -40,120 +40,21 @@ export const pageQuery = graphql`
 
 ---
 
-## Finding an icon
+<Callout>
+  <CalloutTitle as="h4">Self-service icon additions are temporarily unavailable</CalloutTitle>
+  <CalloutText>
+    Due to an ongoing migration from Sketch to Figma, our self-service procedure is currently unavailable. All icon
+    additions to Paste must follow the Github Discussions approach described below.
+  </CalloutText>
+</Callout>
 
 1. Make sure the icon you need doesn’t already exist in [our icon list](/icons).
-2. If the icon can be used for a similar use case to yours in multiple products ("UI icon"), find a new icon by searching through the [Streamline app](https://app.streamlineicons.com/streamline-regular). If the icon needs to be reserved for use in your product only ("logo icon"), please [request one from our Brand team](https://www.twilio.com/brand/resources/logos). 
+2. If the icon can be used for a similar use case to yours in multiple products ("UI icon"), find a new icon by searching through the [Streamline app](https://app.streamlineicons.com/streamline-regular). If the icon needs to be reserved for use in your product only ("logo icon"), please [request one from our Brand team](https://www.twilio.com/brand/resources/logos).
 3. Create a [GitHub discussion](https://github.com/twilio-labs/paste/discussions) with:
    1. A link or screenshot of the icon
```

Comment:
Can these nested GH Discussion steps be a ul instead of an ol? I just don't like the look of nested numbered lists—looks intimidating 😂